src/psyclone/gocean1p0.py

```diff
@@ -782,8 +806,11 @@ def view(self, indent=0):
         '''
         # Generate the upper and lower loop bounds
         # TODO: Issue 440. upper/lower_bound should generate PSyIR
-        self.start_expr = Literal(self._lower_bound(), parent=self)
-        self.stop_expr = Literal(self._upper_bound(), parent=self)
+        self.start_expr = self._lower_bound(parent=self)
```

Comment:
Can the above TODO be removed now seeing as lower and upper bounds do generate PSyIR (albeit with Literal)? The reference to #363 captures future improvement to better support bounds in PSyIR.

Reply:
Yes, good point.